tests/tasks/test_api_list_tasks.py

```diff
@@ -39,8 +39,7 @@ def test_list_tasks_api_second_mentorship_relation(self):
         auth_header = get_test_request_header(self.first_user.id)
         expected_response = marshal(self.tasks_list_2.tasks, list_tasks_response_body)
         actual_response = self.client.get(
-            f"/mentorship_relation/{self.mentorship_relation_w_admin_user.id}"
-            "/tasks",
+            f"/mentorship_relation/{self.mentorship_relation_w_admin_user.id}" "/tasks",
```

Comment:
the tests are not passing due to this and after reformatting the tests passed so the extra two files changed are `tests/tasks/test_api_list_tasks.py` and `tests/users/test_api_list_users.py`